src/mr_reduction/script_output.py

```diff
@@ -97,7 +104,7 @@ def write_tunable_reduction_script(matched_runs, scaling_factors, ipts, output_d
     script += "workspaces =  dict()\n"
     script += "parameters = dict()\n\n"
 
-    search_dirs = list()
```

Comment:
Why "[]" is better than "list()"?

Reply:
https://chatgpt.com/share/e/67576b14-2578-8005-bf96-99a50ee6dcde